guides/07_FAQs.md

```diff
@@ -405,24 +406,18 @@ When the camera LED remains on, it does not necessarily mean the device is broad
 
 ```
 // Select no video device (releases any previously selected device)
-meetingSession.audioVideo.chooseVideoInputDevice(null);
-
-// Stop local video tile (stops sharing the video tile in the meeting)
-meetingSession.audioVideo.stopLocalVideoTile();
-
-// Stop a video preview that was previously started (before session starts)
-meetingSession.audioVideo.stopVideoPreviewForVideoInput(previewVideoElement);
-
-// Stop the meeting session (audio and video)
-meetingSession.audioVideo.stop();
+meetingSession.audioVideo.stopVideoInput();
```

Comment:
For understanding, I saw you remove the example for `stopLocalVideoTile` and `stopVideoPreviewForVideoInput`. I s this because `stopVideoInput()` API handles these automatically?

Reply:
No because those APIs no longer stop the camera streams.